.ci/system-tests/sys_test_common.sh

```diff
@@ -17,32 +19,54 @@ CheckContainerStatus() {
 fi
 }
 
+# clear all policy settings
 setPolicyNone() {
-docker exec $aca_container mysql -u root -D hirs_db -e"Update SupplyChainPolicy set enableEcValidation=0, enablePcAttributeValidation=0, enablePcValidation=0, enableUtcValidation=0, enableFirmwareValidation=0, enableExpiredCertificateValidation=0;" 
+docker exec $aca_container mysql -u root -D hirs_db -e"Update SupplyChainPolicy set enableEcValidation=0, enablePcAttributeValidation=0, enablePcValidation=0,
```

Comment:
Nitpick: space between -e and argument (and following similar lines).

Reply:
addressed with latest [commit](https://github.com/nsacyber/HIRS/pull/430/commits/575be7bd65fa1c924213b77053398776e30662cd)